submissions/Oleksandr-Heleta/popup/index.html

```diff
@@ -0,0 +1,170 @@
+                        <img class="popup_button-img" src="./popup-icons/popup-button.png" alt="popup button">
+                    </label>
+
+
+                    <div class="popup">
+
+                        <input class="popup_more-input" type="checkbox" name="more" id="more_less">
+                        <label class="popup_more_button" for="more_less">
+                            <span id="more">More</span>
+                            <span id="less">Less</span>
+                        </label>
+
+                        <ul class="popup_list">
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/calendar.png" alt="calendar">
+                                    <p>Calendar</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/mail.png" alt="mail">
+                                    <p>Mail</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/news.png" alt="news">
+                                    <p>News</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/maps.png" alt="map">
+                                    <p>Map</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/files.png" alt="files">
+                                    <p>Failes</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/contacts.png" alt="contacts">
+                                    <p>Contacts</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
```

Comment:
Fix icons alignment: just center it
![image](https://user-images.githubusercontent.com/16196199/185042673-be67401d-c496-48e3-bbd9-8c1edaba3473.png)

Reply:
Still not fixed :cry: